tests/test_checker/test_noqa.py

```diff
@@ -23,6 +23,7 @@
     'WPS202',  # since our test case is complex, that's fine
     'WPS204',  # our tests have a lot of overused expressions
     'WPS226',  # we have a lot of ugly strings inside,
+    'WPS353',  # disabled `noqa`comments prevent line from being "blank"
```

Comment:
We should totally not ignore this rule

Reply:
I ran into issues when trying to test this. For example, I added:
```
arr = [
#noqa:WPS353
1]
```

But then,  I start failing the last asset in ```_assert_errors_count_in_output``` when the noqa's are disabled. Could that be because that line contains a COMMENT token and so isn't technically blank anymore?

Reply:
Oh, I see. That's fair. Let it be ignored.